WebHostLib/customserver.py

```diff
@@ -48,12 +48,18 @@ def output(self, text: str):
 
 
 class WebHostContext(Context):
-    def __init__(self):
+    def __init__(self, static_server_data: dict):
+        self.static_server_data = static_server_data
         super(WebHostContext, self).__init__("", 0, "", "", 1, 40, True, "enabled", "enabled", "enabled", 0, 2)
+        del self.static_server_data
```

Comment:
is passing static_server_data where the unpickling happens for multiprocessing?
anyway, i would leave a comment like `# only used in init_datapackage during super().__init__` for the del

Reply:
the pickle barrier is the arguments passed into Process(), the receiving process unpickles them when it starts up.